src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationServiceSpec.scala

```diff
@@ -162,7 +174,7 @@ class MethodConfigurationServiceSpec extends FreeSpec with ScalaFutures with Sca
       "MethodNotAllowed error is returned" in {
         Put(validCopyFromRepoUrl) ~> sealRoute(routes) ~> check {
           status should equal(MethodNotAllowed)
-          responseAs[String] === "HTTP method not allowed, supported methods: GET"
+          errorReportCheck("FireCloud", MethodNotAllowed)
```

Comment:
Orchestration only passes `POST` to Rawls `copyFromMethodRepo` so this check indicates that it's Orchestration returning this error, not Rawls.  See below for the alternate scenario.